src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &__product-image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-title {
+    font-weight: $medium-font;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__product-code {
+    color: $secondary-text-color;
+    margin: 0 0 16px;
+  }
+
+  &__rating-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin: 0 0 24px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    color: $secondary-text-color;
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  &__price-amount {
+    font-weight: $bold-font;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buy-link {
+    height: 40px;
+    border-radius: 5px;
+
+    background-color: $main-color;
+    text-transform: uppercase;
+    text-align: center;
+    text-decoration: none;
+    font-weight: $bold-font;
+    color: $main-contrast-color;
+    font-size: 14px;
+    line-height: 16px;
+
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background-color: $main-contrast-color;
+      border: solid 1px $main-color;
+      color: $main-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
```

Comment:
It seems there is a typo in the selector `&__title`. According to BEM methodology and the rest of your class naming, it should be `&__product-title`. Make sure the class name is consistent with the rest of your BEM naming convention.